src/app/modules/main/chat_section/controller.nim

```diff
@@ -139,6 +144,9 @@ method isCommunity*(self: Controller): bool =
 method getJoinedCommunities*(self: Controller): seq[CommunityDto] =
   return self.communityService.getJoinedCommunities()
 
+method getCommunityById*(self: Controller, communityId: string): CommunityDto =
+  return self.communityService.getCommunityById(communityId)
```

Comment:
I think you can rename this `getCommunity`, since this chat_section represents one community (the community ID is `sectionId`). 
```suggestion
method getCommunityById*(self: Controller): CommunityDto =
  return self.communityService.getCommunityById(self.sectionId)
```

Reply:
done :)